### PR TITLE
add missing community reviewer folders

### PR DESCRIPTION
**Pull request by rrennick**
### Changes proposed in this Pull Request:

This PR adds the `product-editor` and `woocommerce-blocks` folders to the community PR reviewers configuration. Here are two recent community PRs that were not assigned a reviewer:

- [woocommerce-blocks](https://github.com/woocommerce/woocommerce/pull/44799)
- [product-editor](https://github.com/woocommerce/woocommerce/pull/44226)

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. YAML review
2.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: .github/project-community-pr-assigner.yml [hunk 1]
```diff
@@ -85,11 +88,15 @@
...
   - team: ghidorah
-  
+
 "plugins/woocommerce-admin/**/*":
   - team: mothra
   - team: ghidorah
 
+"plugins/woocommerce-blocks/**/*":
+  - team: rubik
+  - team: woo-fse
```
Comment (jonathansadowski) [inline .github/project-community-pr-assigner.yml:98]: yaml looks valid, but we don't have any PRs or issues with this `team: woo-fse` label. Is that being monitored by anyone?

**Review by jonathansadowski — APPROVED** [review]
Thanks for updating Looks good to me. I left a question, but is non-blocking.

File: .github/project-community-pr-assigner.yml (hunk 1 above)
Comment (rrennick) [reply .github/project-community-pr-assigner.yml:98]: Those are Github teams not labels: `woocommerce/rubik` and `woocommerce/woo-fse`.

Comment (jonathansadowski) [reply .github/project-community-pr-assigner.yml:98]: Ahh, right, forgot how this yaml config worked. Thanks. Looks good to me then.

**Pull request merged**